sherpa/fit.py

```diff
@@ -482,7 +510,8 @@ def _get_callback(self, outfile=None, clobber=False):
             warning(e)
 
         self._dep, self._staterror, self._syserror = self.data.to_fit(
-            self.stat.calc_staterror)
+	    self.stat.calc_staterror)
+        self.bkg, junk1, junk2 = self.get_bkg_data( self._dep )
```

Comment:
The 2nd and 3rd arguments will be used for wstat ( data size and response
time, respectively).  They are needed for simultaneous fit of more then 1
data.

On Wed, Aug 26, 2015 at 10:09 AM, Doug Burke <notifications@github.com>
wrote:

> In sherpa/fit.py
> <https://github.com/sherpa/sherpa/pull/90#discussion_r37985690>:
>
> > @@ -482,7 +510,8 @@ def _get_callback(self, outfile=None, clobber=False):
> >              warning(e)
> >
> >          self._dep, self._staterror, self._syserror = self.data.to_fit(
> > -            self.stat.calc_staterror)
> > +	    self.stat.calc_staterror)
> > +        self.bkg, junk1, junk2 = self.get_bkg_data( self._dep )
>
> Can this be changed to
>
> self.bkg, _, _ = self.get_bkg_data(self._dep)
>
> to be explicit that we do not care about the second and third items
> returned by the call?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sherpa/sherpa/pull/90/files#r37985690>.
>